test/integration/clinical/stub-schema.json

```diff
@@ -118,6 +139,1390 @@
                   "polyA+ RNA",
                   "Other RNA fractions"
                 ]
+              },
```

Comment:
what happened here ?

Reply:
the schema was super outdated.... I just copied the stub ver 1.0 from submisison's stub sample-schema.json

Reply:
as discussed I've moved the submission stub-schema to the /test/integration and used in both so shouldn't happen again.